indico/core/signals/event/designer.py

```diff
@@ -11,5 +11,7 @@
 _signals = Namespace()
 
 print_badge_template = _signals.signal('print-badge-template', """
-Called when printing a badge template. The registration form is passed in the `regform` kwarg.
+Called when printing a badge template. 
```

Comment:
This line has trailing whitespace, resulting in the CI failure.

Reply:
Trailing whitespace removed